### PR TITLE
User orders

### PR DESCRIPTION
**Pull request by kmasila00**
ACCOUNT PAGE AND REVIEWS DONE!

### CODE REVIEW

File: browser/js/my-account/my-account.html [hunk 1]
```diff
@@ -10,30 +10,38 @@
...
 							<p>Street Address: <input type="text" class="form-control" value="Street Address"></p>
 							<p>City: <input type="text" class="form-control" value="City"></p>
 							<p>State: <input type="text" class="form-control" value="State"></p>
 							<p>Zip Code: <input type="text" class="form-control" value="Zip Code"></p>
 						</p>
+						<div class="row">
+							<div>
+								<a ng-click="notEditing = !notEditing" class="button medium expanded">Save Changes</a>
+							</div>
+						</div>
```
Comment (jonperrelle) [inline browser/js/my-account/my-account.html:34]: Do the changes persist to the backend?

File: browser/js/my-account/my-account.js [hunk 2]
```diff
@@ -6,15 +6,29 @@ app.config(function ($stateProvider) {
     });
 });
 
-app.controller('myAccountCtrl', function ($scope, Session, OrdersFactory, $stateParams) {
+app.controller('myAccountCtrl', function ($scope, Session, OrdersFactory, $stateParams, ReviewFactory) {
     $scope.userSess = Session
+    let theUser = $stateParams.userId
```
Comment (jonperrelle) [inline browser/js/my-account/my-account.js:11]: change theUser to userId?

File: browser/js/common/directives/navbar/navbar.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state)
         link: function (scope) {
 
             scope.loggedInButtons = [
-                 {label: 'Logout', state: 'logout', auth: true},
+                 {label: 'Logout', state: 'logout', auth: true, },
```
Comment (geoffbass) [inline browser/js/common/directives/navbar/navbar.js]: delete comma?

File: browser/js/my-account/my-account.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+});
+
+app.controller('myAccountCtrl', function ($scope, Session, OrdersFactory, $stateParams, ReviewFactory, UsersFactory) {
+    $scope.userSess = Session
+    console.log(Session);
+    let userId = $stateParams.userId
+
+    OrdersFactory.getUserOrders(userId)
+    .then(function(orders){
+        // console.log(orders);
```
Comment (geoffbass) [inline browser/js/my-account/my-account.js]: delete

File: browser/js/my-account/my-account.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    $stateProvider.state('myAccount', {
+        url: '/my-account/:userId',
+        templateUrl: 'js/my-account/my-account.html',
+        controller: 'myAccountCtrl',
+    });
+});
+
+app.controller('myAccountCtrl', function ($scope, Session, OrdersFactory, $stateParams, ReviewFactory, UsersFactory) {
+    $scope.userSess = Session
+    console.log(Session);
```
Comment (geoffbass) [inline browser/js/my-account/my-account.js]: delete

File: browser/js/my-account/my-account.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        $scope.userWithAddress = user;
+    })
+
+    $scope.reviewing = false;
+
+    $scope.submitReview = function(stars, review, songId){
+        $scope.reviewMade = true;
+        ReviewFactory.createReview(stars, review, songId, userId)
+        .then(function(review){
+            console.log(review)
```
Comment (geoffbass) [inline browser/js/my-account/my-account.js]: delete

File: server/app/routes/users/index.js [hunk 7]
```diff
@@ -79,4 +85,26 @@ router.put('/:id', function (req, res, next) {
...
+router.put('/myAccount/:id', function (req, res, next) {
+  User.findOne({
+    where:{
+      id: req.params.id
+    }
+  })
+  .then(function(user){
+    user.update({firstName:req.body.firstName, lastName:req.body.lastName})
+    .then(function(updatedUser){
+      Address.findOne({
```
Comment (geoffbass) [inline server/app/routes/users/index.js]: return this

File: server/app/routes/users/index.js [hunk 8]
```diff
@@ -79,4 +85,26 @@ router.put('/:id', function (req, res, next) {
   }
 });
 
+router.put('/myAccount/:id', function (req, res, next) {
```
Comment (geoffbass) [inline server/app/routes/users/index.js:97]: needs to send a response

File: server/app/routes/users/index.js [hunk 9]
```diff
@@ -79,4 +85,26 @@ router.put('/:id', function (req, res, next) {
...
+  User.findOne({
+    where:{
+      id: req.params.id
+    }
+  })
+  .then(function(user){
+    user.update({firstName:req.body.firstName, lastName:req.body.lastName})
```
Comment (geoffbass) [inline server/app/routes/users/index.js]: return this

File: server/app/routes/users/index.js [hunk 10]
```diff
@@ -19,7 +20,12 @@ router.get('', function (req, res, next){
 });
 
 router.param('id', function (req, res, next, id) {
-  return User.findById(id)
```
Comment (geoffbass) [inline server/app/routes/users/index.js:36]: return User.findById(id, {include: [Address] } ) ...

File: server/app/routes/users/index.js [hunk 11]
```diff
@@ -79,4 +85,26 @@ router.put('/:id', function (req, res, next) {
   }
 });
 
+router.put('/myAccount/:id', function (req, res, next) {
+  User.findOne({
```
Comment (geoffbass) [inline server/app/routes/users/index.js]: can just use req.user after hitting router.param route

**Pull request merged**